src/main/scala/com/databricks/spark/csv/package.scala

```diff
@@ -15,8 +15,10 @@
  */
 package com.databricks.spark
 
-import org.apache.spark.sql.{SQLContext, DataFrame}
+import org.apache.commons.csv.CSVFormat
+import org.apache.spark.sql.{SQLContext, DataFrame, Row}
 import org.apache.hadoop.io.compress.CompressionCodec
+import org.apache.commons.lang3.StringUtils
```

Comment:
Nit: it is better to sort the imports according to Spark's code style guide: https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Imports